python/pyproject.toml

```diff
@@ -35,6 +35,7 @@ flake8 = { version = "4.0.1", optional = true }
 nbsphinx = { version = "^0.8.8", optional = true }
 boto3 = {version = "^1.22.13", optional = true }
 whylabs-client = {version = "^0.3.0", optional = true}
+mlflow-skinny = {version = "^1.26.1", optional = true}
```

Comment:
minor but this is in the # Doc dependencies section, maybe move it down? We could add a comment for # integrations?

Reply:
will do!